plugins/modules/ios_hsrp_interfaces.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2025 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for ios_hsrp_interfaces
+"""
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ios_hsrp_interfaces
+short_description: Resource module to configure HSRP on interfaces.
+description:
+  - This module provides declarative management of HSRP configuration on interface for Cisco IOS devices.
+version_added: 1.0.0
+author:
+  - Sagar Paul (@KB-perByte)
+notes:
+  - Tested against Cisco IOSXE Version 17.6 on CML.
+  - This module works with connection C(network_cli).
+    See U(https://docs.ansible.com/ansible/latest/network/user_guide/platform_ios.html)
+  - The module examples uses callback plugin (stdout_callback = yaml) to generate task
+    output in yaml format.
+options:
+  config:
+    description: A dictionary of HSP configuration options to add to interface
+    type: dict
```

Comment:
Is there any reason we haven't kept it as list ? IMHO  list of dicts would be more suitable here

Reply:
Definitely a mistake, would correct it. Thanks